docs/BUILDING.md

```
@@ -1,68 +1,84 @@
 # Building AFFiNE Web
 
-> **Warning**:
+> **Warning**
 >
 > This document is not guaranteed to be up-to-date.
 > If you find any outdated information, please feel free to open an issue or submit a PR.
 
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)